docker/kubernetes/create-gameroom-keys.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2018-2020 Cargill Incorporated
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: gameroom-keys
+spec:
+  template:
+    spec:
+      containers:
+      - name: gameroom-keys-generator
+        image: splintercommunity/splinter-cli
+        command:
+          - bash
+        args:
+          - -c
+          - |
+              mkdir -p /keys &&
+              cd keys &&
+              splinter-cli admin keygen alice -q --force &&
```

Comment:
isn't this now `splinter`? or is the `splinter-cli` image using the older CLI name?

Reply:
updated